### DISCUSSION TITLE
Sports field surface

### DISCUSSION THREAD

**Discussion by DennisKempel** [streetcomplete/StreetComplete#2626] [Q&A] [answered]
Hello,relatively many sports grounds in Germany (for example for soccer) have a "Grand" surface, which also seems to be referred to as "ash".
see [https://de.wikipedia.org/wiki/Aschenplatz](https://de.wikipedia.org/wiki/Aschenplatz)

What's the correct choice within SC?

**Comment by westnordost** [comment, marked as answer]
See https://wiki.openstreetmap.org/wiki/Key:surface

I'd say it is surface=clay.
In German StreetComplete translated as "Sandplatz (Lehm)"

**Reply by DennisKempel** [reply]
tks
